tools/wptrunner/wptrunner/browsers/chrome_android.py

```diff
@@ -50,7 +50,10 @@ def executor_kwargs(test_type, server_config, cache_manager, run_info_data,
     capabilities = dict(DesiredCapabilities.CHROME.items())
     capabilities["chromeOptions"] = {}
     # required to start on mobile
-    capabilities["chromeOptions"]["androidPackage"] = "com.google.android.apps.chrome"
+    # Workaround: package name should reflect channel correctly.
+    capabilities["chromeOptions"]["androidPackage"] = "com.android.chrome"
+    # Workaround: ANDROID_SERIAL is ignored in webdriver.
+    capabilities["chromeOptions"]["androidDeviceSerial"] = "emulator-5554"
```

Comment:
What is `emulator-5554`? Is this just happen to be the device ID for your emulator that you were running? I don't remember having to set this last time. Is it only required when there are multiple devices connected?

Reply:
You're correct. I'll remove this part as this only concerns multiple device scenarios. chromedriver should use ANDROID_SERIAL to detect the current device, which it doesn't at the moment, and it should be fixed in chromedriver instead.

Reply:
https://cs.chromium.org/chromium/src/chrome/test/chromedriver/chrome_launcher.cc?l=591&rcl=bd218f82c33a10cdf226840ce28c597080ceb558
IIRC, ChromeDriver simply uses a bunch of `adb` commands underneath for finding devices etc., so it should be an easy fix.

Reply:
Thanks for the suggestion. Landing this CL: https://chromium-review.googlesource.com/c/chromium/src/+/1652556

Reply:
Removed now.